src/components/webView.rei

```diff
@@ -1,8 +1,11 @@
 type source;
 
-let source:
```

Comment:
This is a breaking change. We can make it non breaking by aliasing `source` to `sourceUri` and marking it as deprecated

Reply:
Good point, fixed in dec94b7